packages/design-system/src/components/IdleTimeout/IdleTimeout.test.tsx

```diff
@@ -118,18 +118,46 @@ describe('Idle Timeout', () => {
     });
   });
 
+  it('should reset timeouts if timeToTimeout changes', () => {
+    const { rerender } = renderIdleTimeout();
+    rerender(<IdleTimeout {...defaultProps} timeToTimeout={10} />);
+    expect(localStorage.setItem).toHaveBeenCalledTimes(3);
+  });
+
+  it('should reset timeouts if timeToWarning changes', () => {
+    const { rerender } = renderIdleTimeout();
+    rerender(<IdleTimeout {...defaultProps} timeToWarning={4} />);
+    expect(localStorage.setItem).toHaveBeenCalledTimes(3);
+  });
+
+  it('should update message time if timeToTimeout changes', () => {
+    const formatMessage = jest.fn();
+    const { rerender } = renderIdleTimeout({ formatMessage });
+    rerender(<IdleTimeout {...defaultProps} timeToTimeout={10} formatMessage={formatMessage} />);
+    showWarning();
+    expect(formatMessage).toHaveBeenNthCalledWith(1, 7);
+  });
+
+  it('should update message time if timeToWarning changes', () => {
+    const formatMessage = jest.fn();
+    const { rerender } = renderIdleTimeout({ formatMessage });
+    rerender(<IdleTimeout {...defaultProps} timeToWarning={4} formatMessage={formatMessage} />);
+    showWarning(1644051720); // start time + (4 * 60000ms) (4 minutes)
+    expect(formatMessage).toHaveBeenNthCalledWith(1, 1);
+  });
+
   it('default formatMessage should replace time in message', () => {
     const { getByRole } = renderIdleTimeout();
     showWarning();
     const dialogBodyText = getByRole('main');
     expect(dialogBodyText.textContent).toEqual(
-      `You've been inactive for a while.Your session will end in 3 minutes.Select "Continue session" below if you want more time.`
+      `You've been inactive for a while.Your session will end in 2 minutes.Select "Continue session" below if you want more time.`
     );
   });
 
   it('default formatMessage should adjust message for singular minute vs multiple', () => {
-    const { getByRole } = renderIdleTimeout({ timeToWarning: 1 });
-    showWarning(1644051720);
+    const { getByRole } = renderIdleTimeout({ timeToWarning: 4 });
+    showWarning(1644051720); // start time + (4 * 60000ms) (4 minutes)
```

Comment:
it might make this easier to follow if we actually used that math here instead of this raw number